apps/system/js/hierarchy_manager.js

```diff
@@ -58,6 +58,7 @@
      * @type {Array}
      */
     PRIORITIES: [
+      'OverlayWindowManager',
```

Comment:
I don't think the screen reader issue is a problem for us. If a helpee can't see the screen, the helper will not point parts of the screen and just use text or voice to guide the helpee.